docs/UserGuide.md

```diff
@@ -176,17 +235,60 @@ If you are using a PDF version of this document, be careful when copying and pas
+specific to those commands.
+
+| Action                                                                   | Format                                                                                                                   | Examples                                                              |
+|--------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------|
+| **[Add appointment](#adding-an-appointment-add-appt)**                   | `add appt i/PERSON_ID d/DATE_TIME ty/APPOINTMENT_TYPE [s/SICKNESS] [m/MEDICINE]`                   <br/>           <br/> | `add appt i/1 d/2024-12-01 09:30 ty/Consulation s/Diabetes m/Insulin` |
+| **[List all appointments](#listing-all-appointments-list-appt)**         | `list appt`                                                                                                              | `list appt`                                                           |
+| **[Edit appointment](#editing-an-appointment-edit-appt)**                | `edit appt INDEX [i/PERSON_ID] [d/DATE_TIME] [ty/APPOINTMENT_TYPE] [s/SICKNESS] [m/MEDICINE]`                            | `edit appt 3 d/2024-12-05 13:00 m/Panadol`                            |
+| **[Find appointment](#finding-appointments-by-name-and-date-find-appt)** | `find appt KEYWORD [MORE_KEYWORDS]`                                                                                      | `find appt d/2024-12-05`                                              |
+| **[Delete appointment](#deleting-an-appointment-delete-appt)**           | `delete appt INDEX`                                                                                                      | `delete appt 2`                                                       |
+| **[Clear all appointments](#clearing-all-appointments-clear-appt)**      | `clear appt`                                                                     <br/>                                   | `clear appt`                                                          |
+
+<br>
+
+---
 
-The following sections describe the various commands you can use in the DocTrack application:
+<a href="#general-commands" class="badge bg-dark">General commands</a>
+Click this section to know more about the commands, or click on the individual commands below for information
+specific to those commands.
 
-- [Person Commands](#person-commands)
-- [Appointment Commands](#appointment-commands)
-- [General Commands](#general-commands)
+| Action                                | Format | Examples |
+|---------------------------------------|--------|----------|
+| **[Help](#viewing-help-help)**        | `help` | `help`   |
+| **[Exit](#exiting-the-program-exit)** | `exit` | `exit`   |
 
 <br>
 
-### Person Commands
+---
+
+### **Person Commands**
```

Comment:
For each command, should we add a navigation link for user to navigate back up? The coloured background would be nice! It would be useful especially as a PDF format.